src/components/DatasetCard.vue

```diff
@@ -167,7 +167,6 @@ export default {
     },
     propogateCardAction: function(action){
       EventBus.$emit("PopoverActionClick", action)
-      this.$emit('contextUpdate', action)
```

Comment:
Thanks for removing that but SidebarContent hasgot an event handler on line 37.

Reply:
tyty I went ahead and removed all the context card variables and references